config/v1/types_dns.go

```diff
@@ -48,6 +48,12 @@ type DNSSpec struct {
 	//
 	// +optional
 	PrivateZone *DNSZone `json:"privateZone,omitempty"`
+	// upstreams is a list of DNS resolvers to provide name query delegation for a base domain.
```

Comment:
s/upstreams/Upstreams

Reply:
Counter to Go conventions, the downcasing is actually correct in this case for OpenAPI generation. Silly but true